### ISSUE TITLE
Verifying key re-request does not work.

### ISSUE THREAD

**Issue by Qwertie-** [vector-im/riot-android#2590]
I have been able to reproduce this a few times for a while now.

# Steps to reproduce
1. Have riot web already signed in
2. Sign in to riot on a new android device
3. From android, click on "Re-request encryption keys" on an encrypted message sent before signing in on mobile
4. A popup will open on web asking if you want to verify or share without verification.
5. Click start verification on web

After this nothing happens on mobile. Clicking send without verification decrypts the message on mobile.

**Comment by bmarty** [comment]
Hi @Qwertie- , in normal usage case, step 3 is not necessary. Riot-Android already requests the encryption key automatically to any other connected client with the same account.
The "re-request" is only useful if you have dismiss the popup on the Riot web client.

That said, at step 5, you have to click on the next popup on 'I verified that the key match'. If it's not working, it's maybe a problem on Riot-Web then. If you clicked on 'Send without verification' and the message is decrypted on Mobile, there is no issue then? Or I misunderstand something, please let me know

**Comment by Qwertie-** [comment]
Riot does automatically request the key for web but I don't see anything on my phone so I can't confirm that it was my phone that sent the request.

**Comment by bmarty** [comment]
Yes, use has no feedback on automatic request, except on the other clients, and of course when the messages get decrypted.
You can confirm that your phone has sent the request by comparing the device key.

Are you unable to decrypt your messages?

**Comment by Qwertie-** [comment]
Ah ok. Looks like it's probably working as intended then. Maybe a slight UX issue as I wasn't sure what I was meant to compare with when the key was only shown on one of my devices.

**Comment by bmarty** [comment]
The UX issue is (partially) tracked here: #2487
